src/query/interpreter.cpp

```diff
@@ -3165,6 +3306,9 @@ void Interpreter::Abort() {
   if (!db_accessor_) return;
 
   db_accessor_->Abort();
+  for (auto &qe : query_executions_) {
```

Comment:
Could we just call `query_executions_.clear()`?

Reply:
cc @Darych  but I think no because we support prepared queries or something like this so we have to use it or something. There is a good reason for that I am sure.